src/propslib/DelegationConfig.cpp

```diff
@@ -225,7 +231,8 @@ try
     // We also find the default here while we have the list of available ones so
     // we can return the opaque structure instead of the raw IID.
     IID defCon;
-    LOG_IF_FAILED(s_GetDefaultConsoleId(defCon));
+    bool isTerminalAsDefault;

```

Comment:
`isTerminalAsDefault` is false here, right?

Reply:
No, by default primitive types have an undefined value before they're initialized.
But `s_GetDefaultConsoleId` will always set this variable no matter what.